pkg/test/nodeclass.go

```diff
@@ -54,9 +62,13 @@ func AKSNodeClass(overrides ...v1alpha2.AKSNodeClass) *v1alpha2.AKSNodeClass {
 }
 
 func ApplyDefaultStatus(nodeClass *v1alpha2.AKSNodeClass, env *coretest.Environment) {
```

Comment:
Do all these need to be exported?

Reply:
All the ones that are exported I think should be